<launch>
  <param name="njoy" type="int" value="2"/>
  <node pkg="joy" type="joy_node" name="joy1">
    <remap from="joy" to="joy/1"/>
    <param name="dev" type="string" value="/dev/input/js0"/>
    <param name="deadzone" value="0.0" />
    <param name="autorepeat_rate" value="100.0" type="double"/>
  </node>
  <node pkg="joy" type="joy_node" name="joy2">
    <remap from="joy" to="joy/2"/>
    <param name="dev" type="string" value="/dev/input/js1"/>
    <param name="deadzone" value="0.0" />
    <param name="autorepeat_rate" value="100.0" type="double"/>
  </node>
  <node pkg="multijoy" name="multijoy_node" type="multijoy_node.py" output="screen"/>
</launch>
